api.md

```diff
@@ -138,7 +145,7 @@ Methods:
 Types:
 
 ```python
-from finch.types import Provider
+from finch.types import BenefitSupportType, Provider
```

Comment:
Let's scope down changes for this release to only include: 

• (new endpoint) /forward
• (new response field) source_id on /employment

---

So let's exclude this change.